docs/OV_Runtime_UG/precision_control.md

```diff
@@ -0,0 +1,31 @@
+# Precision Control {#openvino_docs_OV_UG_Precision_Control}
+
+@sphinxdirective
+
+Regardless of IR precision, devices will run in high performance mode by default. For GPU this means ``fp16`` inference and for CPU - ``bf16`` inference (if available). Previously, you had to convert the IR to ``fp16`` for the GPU to run in ``fp16``, and for the CPU, it was high precision by default. Now the devices have been aligned and this selection has been disconnected from IR precision. If high performance is causing the accuracy issue (only seen a few times in history), you can use the ``inference_precision`` hint and set it to accuracy.
+
+Separately, you can control the IR precision. Essentially the IR precision becomes a way to compress your model by reducing the precision of the weights, and it doesn't affect how devices execute the model.
+
+This change clears up a lot of confusion where, for example, you couldn't execute a high-performance model on the GPU by default, and the behavior between devicess was different.
+
+.. note::
+
+   All devices (except GNA) only support floating-point data types (``f32``, ``f16``, ``bf16``) as a value for ``inference_precision`` attribute, because quantization cannot be done in Runtime. The GNA plugin has the ability to perform model quantization on ``core.compile_model()`` call, so it supports integer data types in addition to ``f32``.
+
+Execution Mode
+##############
+
+``ov::hint::execution_mode`` is a high-level hint to control whether the user wants to keep the best accuracy (**ACCURACY mode**) or if the plugin can do some optimizations that may lower the accuracy for performance reasons (**PERFORMANCE mode**)
```

Comment:
Let's add more details.
In ACCURACY mode plugin is not allowed to do conversions of floating point tensors to smaller floating point type, so the plugins tries to keep accuracy metrics as close as possible to the original values received after training with respect to the actual device capabilities. That means that most of the plugins will do inference  in fp32 precision if the device supports it. 
In PERFORMANCE mode plugin may do conversion to smaller data types and apply some other optimizations which may have certain impact on the accuracy metrics, though we still try to minimize accuracy drop and in some cases may use mixed precision execution.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/17413/commits/42249732907664d595498e427f8288c168c70e4c